src/error.rs

```diff
@@ -67,8 +67,8 @@ pub enum EbpfError<E: UserDefinedError> {
     #[error("virtual address overlap {0:x?}")]
     VirtualAddressOverlap(u64),
     /// Access violation
-    #[error("out of bounds memory {1:?} (insn #{0}), addr {2:#x}/{3:?} \n{4}")]
-    AccessViolation(usize, AccessType, u64, u64, String),
+    #[error("Access violation in {4} section at address {2:#x} of size {3:?} by instruction #{0}")]
```

Comment:
It would be a guess and maybe more confusing but it might be helpful to report which stack frame was violated (or between which two)

Reply:
One can tell from the vm_addr already, right?
And also, if we were to display a frame index it would be present in all sections, not just the stack.

Reply:
It would have to be a new error type that is specific to stack frame violations